backends/edk2-compat/edk2-svc-generate.c

```diff
@@ -139,44 +53,103 @@ int performGenerateCommand(int argc,char* argv[])
 	size_t outBuffSize, size;
 	struct hash_funct *hashFunction;
 	unsigned char *buff = NULL, *outBuff = NULL;
+	char *programName = NULL;
 	struct Arguments args = {	
 		.helpFlag = 0, .inpValid = 0, .signKeyCount = 0, .signCertCount = 0, .alreadySignedFlag = 2,
```

Comment:
I got super confused by the meaning of `alreadySignedFlag` later on in this file. A good followup patch would be to replace it with an enum of some sort.

Reply:
Good idea, I am going to through that into a new commit in this PR.